Update building-a-native-android-app.md (#845)
codemagic-ci-cd/codemagic-docs: content/yaml-quick-start/building-a-native-android-app.md

    max_build_duration: 60
    instance_type: mac_mini
    environment:
      groups:

>         - keystore_credentials # Uncomment this if you're not using code-signing-identities <-- Includes - CM_KEYSTORE, CM_KEYSTORE_PASSWORD, CM_KEY_PASSWORD, CM_KEY_ALIAS

Reviewer: ```suggestion
        - keystore_credentials # Comment this out if you are using code-signing-identities <-- Includes - CM_KEYSTORE, CM_KEYSTORE_PASSWORD, CM_KEY_PASSWORD, CM_KEY_ALIAS
```
